services/jenkins/jenkins-coverage.tester.js

```diff
@@ -9,19 +9,17 @@ const t = (module.exports = require('../tester').createServiceTester())
 // https://wiki.jenkins.io/pages/viewpage.action?pageId=58001258
 
 t.create('jacoco: job found')
-  .get('/jacoco/https/wso2.org/jenkins/view/All%20Builds/job/archetypes.json')
-  .timeout(10000)
+  .get('/jacoco/https/builds.apache.org/job/Derby-JaCoCo.json')
```

Comment:
I'm hesitant to change this one.  We've had recurring issues with the builds.apache.org instance in the past, and it kept timing out for me when I was working on #3337. Obviously the builds.apache.org instance is working at the moment but it's been way too flaky in the past.  It doesn't look like this particular test was failing/timing out, so I think we might be better off keeping this test with the wso2.org instance. 

the ubuntu and updates.jenkins-ci.org instances (including the ones below) are definitely problematic and need to be replaced, but I'd suggest we find a target jenkins instance other than builds.apache.org. 

What do you think?

Reply:
Fair enough, I've removed all builds.apache.org as well a few of the ubuntu and updates.jenkins-ci.org. The remaining ones seem to be working fine for now. 👍